ui/apps/platform/src/Containers/AccessControl/AuthProviders/RuleGroups.tsx

```diff
@@ -172,7 +172,8 @@ function RuleGroups({
                                             <TrashIcon />
                                         </Button>
                                     </FlexItem>
-                                ) : (
+                                )}
+                                {!isUserResource(group?.props?.traits) && (
```

Comment:
Ivan, does this need `!disabled && !isUserResource(group?.props?.traits) && (`

The `disabled` prop (versus `isDisabled` function) gets value `disabled={isViewing}` in AuthProviderForm

Reply:
Maybe I miss something, but I don't understand now why we need to include `disabled` here. 

The incorrect logic before:
1. If not disabled for whatever reason, show delete button
2. Otherwise show declarative info

The correct(I hope) logic now:
1. If not disabled for whatever reason, show delete button.
2. If disabled because resource is non-user, show declarative info

Reply:
Aha, I read the following sentence too quickly and saw only **in view mode** but missed the implied **and**.

> information message when user resource is in view mode.

An observation (not a change request) is the information consists of 2 pieces of information:
* This rule is managed declaratively. [I added period to simulate how I read it.]
* It can only be edited declaratively. [My mental fail was, not relevant in view mode, therefore do not render, forgetting that the first point is always relevant for declarative resource.]

The second point does bring up a question: The changed code has conditional rendering for User versus Declarative. Remind me, there is no System auth provider?

Reply:
There are no `System` groups or auth providers. The only possible `System` resources are roles, permission sets and access scopes